src/docker-osx-dev

```diff
@@ -440,11 +444,24 @@ function init_docker_host {
 
 #
 # Installs rsync on the Boot2Docker VM, unless it's already installed.
+# If the main repository is down, rsync will be installed from a mirror
+# If the mirror is down, download rsync binary and place in Boot2Docker VM
 #
 function install_rsync_on_docker_host {
   log_info "Installing rsync in the Docker Host image"
 
-  $DOCKER_HOST_SSH_COMMAND "if ! type rsync > /dev/null 2>&1; then tce-load -wi rsync; fi"
+  local readonly MIRROR="http://distro.ibiblio.org/tinycorelinux/6.x/x86_64/tcz"
+  local readonly DIR="/tmp/tce/optional/"
+  local readonly ACL="acl.tcz"
```

Comment:
I just found this:
https://github.com/puppetlabs/Razor-Microkernel/wiki/Setting-up-a-TCE-repository---mirror

Apparently tce-load looks at /opt/tcemirror to determine which repository to load tcz files from. A simple "echo 'http://distro.ibiblio.org/' > /opt/tcemirror" should set the mirror for tce-load to properly install rsync and its dependencies automatically.